processors/rot13.go

```diff
@@ -0,0 +1,60 @@
+package processors
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ROT13 converts string with ROT13 cypher.
+// https://en.wikipedia.org/wiki/ROT13
+type ROT13 struct{}
+
+func (p ROT13) Name() string {
+	return "rot13"
+}
+
+func (p ROT13) Alias() []string {
+	return []string{"rot13-encode", "rot13-decode", "rot13-enc", "rot13-dec"}
```

Comment:
I'm a little bit confused about the alias now, should we keep `rot13-decode` and `rot13-dec` or not in the list?  I think I might approve but I need to think little bit more on this

Reply:
I added them because some people may think they could decode this way. I didn't want to remove the rot13-enc* aliases, I thought it would be good to add them for people who don't know how it works.

I would say it's up to you. I can remove them, remove also the rot13-enc*. You tell me

Reply:
I would like to keep `rot13-encode` and `rot-enc` and remove `"rot13-decode"` `"rot13-dec"`.

Reply:
pushed back